### PR TITLE
Gracefully stop yarn container

### PR DESCRIPTION
**Pull request by mzmssg**
An enough graceful timeout(65s) for yarn container exit

### CODE REVIEW

File: src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml.template [hunk 1]
```diff
@@ -350,10 +350,15 @@ jobs are started with the DockerContainerExecutor.
...
+  </description>
 </property>
-<description>
-     Image command before launch_container script.
-</description>
-  
+
+  <property>
+    <name>yarn.nodemanager.sleep-delay-before-sigkill.ms</name>
+    <value>65000</value>
```
Comment (abuccts) [inline src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml.template]: why 65s?

File: src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml.template [hunk 2]
```diff
@@ -350,10 +350,15 @@ jobs are started with the DockerContainerExecutor.
 
 <property>
   <name>yarn.nodemanager.docker-container-executor.script-command</name>
-  <value>cp -r /docker/* /usr/bin/  &amp;&amp; cp /hadoop-configuration-for-jobs/* $HADOOP_CONF_DIR/ </value>
+  <value> trap &quot; &quot; TERM; cp -r /docker/* /usr/bin/  &amp;&amp; cp /hadoop-configuration-for-jobs/* $HADOOP_CONF_DIR/ </value>
```
Comment (abuccts) [inline src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml.template]: trailing spaces?

File: src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml.template (hunk 1 above)
Comment (mzmssg) [reply src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml.template]: No special meaning, just pick from #2408

File: src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml.template (hunk 2 above)
Comment (mzmssg) [reply src/hadoop-node-manager/deploy/hadoop-node-manager-configuration/yarn-site.xml.template]: No

**Pull request merged**